File: 1.Layers/1.3Application/1.3.2MainModule/Application.MainModule/DTOs/Mobile/Cortes/VentasCorteDTO.cs
/**
 *  VentasCorteDTO
 *  Modelo DTO para la lista de ventas de las cuales se ligara el
 *  corte de caja
 *  Developer: Jorge Omar Tovar Martínez <jorge.tovar@neoteck.com.mx>
 *  Company: Neoteck
 *  Date: 04/12/2018
 *  Update 04/12/2018 
 */
namespace Application.MainModule.DTOs.Mobile.Cortes
{
    public class VentasCorteDTO
    {
        public string Corte { get; set; }
        public string TiketVenta { get; set; }
        public int IdVenta { get; set; }
    }
}

File: 1.Layers/1.1Presentation/1.1.2Mobile/app/src/main/java/com/neotecknewts/sagasapp/Model/VentasCorteDTO.java
/*
 * VentasCorteDTO
 * Permite almacenar todos las ventas que se les este realizando el corte de
 * caja para su actualización en el servicio y asignar el código de corte a la venta
 * @author Jorge Omar Tovar Martínez jorge.tovar@neoteck.com.mx
 * @company Neoteck
 * @Date 05/12/2018
 * @Update 05/12/2018
 */
package com.neotecknewts.sagasapp.Model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class VentasCorteDTO implements Serializable {
    @SerializedName("Corte")
    private String Corte;

    @SerializedName("TiketVenta")
    private String TiketVenta;

    @SerializedName("IdVenta")
    private int IdVenta;

    public String getCorte() {
        return Corte;
    }

    public void setCorte(String corte) {
        Corte = corte;
    }

    public String getTiketVenta() {
        return TiketVenta;
    }

    public void setTiketVenta(String tiketVenta) {
        TiketVenta = tiketVenta;
    }

    public int getIdVenta() {
        return IdVenta;
    }

    public void setIdVenta(int idVenta) {
        IdVenta = idVenta;
    }
}
